rtil/src/threads/rebo/rebo_init.rs

```diff
@@ -704,6 +709,10 @@ fn get_level() -> i32 {
 fn set_level(level: i32) {
     LevelState::set_level(level);
 }
+#[rebo::function("Tas::set_end_partial")]
+fn set_end_partial(ep: f32) {
+    LevelState::set_end_partial(ep);
+}
```

Comment:
If you add all 4 functions to `LevelState`, why only expose one of them?

Reply:
Good point. Will fix.